src/components/pages/search.less

```diff
@@ -48,3 +48,34 @@
   flex-wrap: wrap;
   justify-content: flex-start;
 }
+
+.SearchPage-cellTagsExpandLink {
+  font-size: 0.9em;
+  display: inline-block;
+  margin-left: 0.2em;
+  height: 20px;
+  padding: 3px;
+  margin: 3px;
+  margin-left: 1em;
```

Comment:
The margins here are a bit confusing, `margin-left` is defined twice and there's a mix of margins in `px` and `em` 🤔

Reply:
I replaced it anyway :-)

Reply:
The first `margin-left` could still be removed, since you set `margin` afterwards, right?

Reply:
OMG, yes, of course. Removed it!